test/python/ibmq/test_ibmq_qobj.py

```diff
@@ -13,88 +13,99 @@
 
 import os
 import unittest
+import functools
 from qiskit import (ClassicalRegister, QuantumCircuit, QuantumRegister, compile)
-
 from qiskit import IBMQ, Aer
 from qiskit.qasm import pi
-from ..common import requires_qe_access, JobTestCase, slow_test
+
+from ..common import require_multiple_credentials, JobTestCase, slow_test
+
+# Timeout duration
+TIMEOUT = int(os.getenv("IBMQ_TESTS_TIMEOUT", 10))
+BLACKLIST = [bl.strip() for bl in
```

Comment:
We use environment variables to keep credentials secret but I think there is no need of keeping the blacklist secret. Can we make the variable an explicit list defined here?

Reply:
Ok, the same for TIMEOUT?